sql/wsrep_mysqld.cc

```diff
@@ -2228,6 +2305,9 @@ static int wsrep_TOI_event_buf(THD *thd, uchar **buf, size_t *buf_len) {
     case SQLCOM_DROP_TABLE:
       err = wsrep_drop_table_query(thd, buf, buf_len);
       break;
+    case SQLCOM_GRANT:
```

Comment:
Does this bug affect REVOKE queries as well?

Reply:
`mysql_grant` rewrites query only for GRANT. 
REVOKE does not have AS clause, nothing about implicit curren_user, so not affected.